src/components/table/table.jsx

```diff
@@ -38,7 +37,7 @@ Table.propTypes = {
 };
```

Comment:
PropType for minWidth need to be PropTypes.oneOfType([PropTypes.number, PropTypes.string]),

Reply:
Sure! I fixed it